examples/lock-app/lock-common/lock-app.zap

```diff
@@ -5372,6 +5372,54 @@
               "maxInterval": 65344,
               "reportableChange": 0
             },
+            {
+              "name": "DoorOpenEvents",
```

Comment:
Is this now just mocking a device that never has the door opened? Is there any test that door open events does anything?